client.go

```diff
@@ -1143,10 +1099,10 @@ func (c *Client[TTx]) ID() string {
 	return c.config.ID
 }
 
-func insertParamsFromArgsAndOptions(args JobArgs, insertOpts *InsertOpts) (*dbadapter.JobInsertParams, error) {
+func insertParamsFromArgsAndOptions(args JobArgs, insertOpts *InsertOpts) (*riverdriver.JobInsertParams, *dbunique.UniqueOpts, error) {
```

Comment:
Haha, yeah it felt to me like it works okay here, but I'm amenable to whatever. This one will be refactorable since it's all internal if you want to take another shot at it.